widget/textgrid.go

```diff
@@ -114,7 +114,7 @@ func (t *TextGrid) Text() string {
 		count += len(row.Cells)
 	}
 
-	runes := make([]rune, count)
+	runes := make([]rune, int(math.Max(0, float64(count))))
```

Comment:
It would probably be easier to read and a bit cleaner if it's just:
```go
if count == 0 {
    return ""
}
```
No need for math or to fall through the for loops. We like early exits for corner cases :)